design-pages/alpacania.html

```diff
@@ -14,146 +14,222 @@
 
 
 	<body id="internal-page">
-		<div id="header">
-			<div id="content">
-				<div id="hamburger">
+
+        <div id="header">
+            <div class="content">
+                <div id="hamburger">
                     <img src="/images/buttons/hamburger.png" />
                 </div>
                 
+                <div id="logo">
+                    <a href="/index.html">
+                        <img src="/images/buttons/logo-name.png" />
+                    </a>
+                </div> <!-- logo end -->
+            
+                <div id="nav">
+                    <ul>
+                        <li><a href="/index.html">Work</a></li>
+                        <li><a href="/about.html">About</a></li>
+                        <li><a href="https://www.facebook.com/xzhangstudio/" target="_blank"><img src="/images/buttons/button_facebook_dark.png" /></a></li>
+                        <li><a href="https://www.instagram.com/xzhangstudio/" target="_blank"><img src="/images/buttons/button_inistagram_dark.png" /></a></li>
+                    </ul>
+                </div> <!-- nav -->
+            </div> <!-- content -->
+        </div> <!-- header end -->
+
 
-				<div id="logo">
-					<a href="/index.html"><img src="/images/buttons/logo-img.png" /><img src="/images/buttons/logo-name.png" /></a>
-				</div>
+		<div id="gallery">
 
-				<div id="nav">
-					<ul>
-                        <li><a href="/illustration.html">illustration</a></li>
-                        <li><a href="/design.html">design</a></li>
-                        <li><a href="/about.html">about</a></li>
-                        <li><a href="https://www.facebook.com/xzhangstudio/" target="_blank"><img src="/images/buttons/button_facebook_light.png" /></a></li>
-                        <li><a href="https://www.instagram.com/xzhangstudio/" target="_blank"><img src="/images/buttons/button_inistagram_light.png" /></a></li>
-					</ul>
-				</div>
-			</div> <!-- content -->
+			<h1><span class="main-title">Alpacania | </span>
+			<span class="sub-title">Design, game</span></h1>
+			<div id="line"></div>
```

Comment:
the `<div id="line></div>` is not necessary. add a `border-top` to your `.thumbnavi-block`